Shared/Objects/UserPermissions.swift

```diff
@@ -20,11 +20,24 @@ struct UserPermissions {
 
     struct UserItemPermissions {
 
+        @StoredValue(.User.enableItemDeletion)
```

Comment:
Do not use these property wrappers in this way. Instead, get the values via the subscript in the `init`.